package-lock.json

```diff
@@ -12,6 +12,7 @@
 			"version": "1.10.1",
 			"resolved": "https://registry.npmjs.org/@discordjs/builders/-/builders-1.10.1.tgz",
 			"integrity": "sha512-OWo1fY4ztL1/M/DUyRPShB4d/EzVfuUvPTRRHRIt/YxBrUYSz0a+JicD5F5zHFoNs2oTuWavxCOVFV1UljHTng==",
+			"license": "Apache-2.0",
```

Comment:
Did you run an `npm ci`? Looks like some package versions are being bumped. I have the details of `Apache-2.0` memorized, so we should probably check if we out of compliance with them at some point (I think we're fine in terms of `MIT`).